components/shared/fonts/font_template.rs

```diff
@@ -155,7 +163,7 @@ pub struct FontTemplate {
     /// This is not serialized, as it's only useful in the [`super::FontContext`]
     /// that it is created in.
     #[serde(skip)]
-    pub stylesheet: Option<DocumentStyleSheet>,
+    font_face_rule: Option<FontFaceRuleInitiator>,
```

Comment:
Please store these two types separately, as I think they both deserve their own fields and rustdoc. The rustdoc here is now out of date, but wouldn't be if you just add another member called `font_face_rule`.